### ISSUE TITLE
Examples don't work as shown, due to Jupyter "Blocking request with no referer"

### ISSUE THREAD

**Issue by maciejb** [mapbox/mapboxgl-jupyter#165]
I tried running the examples as shown in the docs, e.g. the CircleViz in the README, and the result is a blank map (tiles but no markers). 

Looking at the Jupyter logs:
```
[I 17:19:42.806 NotebookApp] 302 GET /notebooks/points.geojson (::1) 1.54ms
[W 17:19:42.821 NotebookApp] Blocking request with no referer
[W 17:19:42.822 NotebookApp] 403 GET /files/points.geojson (::1): Blocking request from unknown origin
[W 17:19:42.824 NotebookApp] 403 GET /files/points.geojson (::1) 2.88ms referer=None
```

This can easily be worked around by storing the `df_to_geojson()` to a variable instead of a file, and reading that variable when creating the viz.

Tried with a few different browsers and both a local and remote notebook server. Is this an issue that can be resolved in the mapboxgl module, or should the documentation be changed?

**Comment by ryanbaumann** [comment]
We should update the documentation to use an in-memory variable assignment as opposed to a flat file export.  The web servers that run various Jupyter environments have various configurations (`cors` settings, relative file paths, etc) that can cause issues with referencing a locally exported file.  

Thanks for flagging this @maciejb

**Comment by almo-ltoeroek** [comment]
same here

**Comment by ryanrolds** [comment]
Encountering this too, I was only able to get it to work with `c.NotebookApp.disable_check_xsrf = True`, which is *not recommended* for security reasons.

**Comment by nygeog** [comment]
Launching notebook `jupyter notebook --NotebookApp.disable_check_xsrf=True` also worked for me.

**Comment by ryanbaumann** [comment]
Closed with PR #170
